motors/tacho_motor_class.c

```diff
@@ -255,7 +251,7 @@ struct tacho_motor_state_item {
 
 static struct tacho_motor_value_names tacho_motor_states[NUM_TM_STATES] = {
 	[TM_STATE_RUNNING]	= { "running" },
-	[TM_STATE_RAMPING]	= { "ramping" },
+	[TM_STATE_STOPPED]	= { "stopped" },
```

Comment:
You should add a section to the docs about motor state. I think we need to hash out the status flags a bit more. 

Here is my idea of what "running" and "ramping" mean. I also added "stopping" which could be useful.

![selection_002](https://cloud.githubusercontent.com/assets/963645/12955371/4deddd56-cfe8-11e5-842a-4559a490486a.png)

I also think the "overloaded" state from "Motor Status" section in <https://github.com/ev3dev/ev3dev/issues/282#issue-59308478> could be useful. And maybe a flag to indicate if passive braking is enabled (we already have a "holding" flag for active braking).